### PR TITLE
i18n: update french translation fr-FR

### PR DESCRIPTION
**Pull request by superzazu**
# Pull Request Checklist

### Note to first-time contributors: Please open a discussion post in [Discussions](https://github.com/open-webui/open-webui/discussions) and describe your changes before submitting a pull request.

**Before submitting, make sure you've checked the following:**

- [x] **Target branch:** Please verify that the pull request targets the `dev` branch.
- [x] **Description:** Provide a concise description of the changes made in this pull request.
- [x] **Changelog:** Ensure a changelog entry following the format of [Keep a Changelog](https://keepachangelog.com/) is added at the bottom of the PR description.
- [x] **Documentation:** Have you updated relevant documentation [Open WebUI Docs](https://github.com/open-webui/docs), or other documentation sources?
- [x] **Dependencies:** Are there any new dependencies? Have you updated the dependency versions in the documentation?
- [x] **Testing:** Have you written and run sufficient tests for validating the changes?
- [x] **Code review:** Have you performed a self-review of your code, addressing any coding standard issues and ensuring adherence to the project's coding standards?
- [x] **Prefix:** To cleary categorize this pull request, prefix the pull request title, using one of the following:
  - **BREAKING CHANGE**: Significant changes that may affect compatibility
  - **build**: Changes that affect the build system or external dependencies
  - **ci**: Changes to our continuous integration processes or workflows
  - **chore**: Refactor, cleanup, or other non-functional code changes
  - **docs**: Documentation update or addition
  - **feat**: Introduces a new feature or enhancement to the codebase
  - **fix**: Bug fix or error correction
  - **i18n**: Internationalization or localization changes
  - **perf**: Performance improvement
  - **refactor**: Code restructuring for better maintainability, readability, or scalability
  - **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)
  - **test**: Adding missing tests or correcting existing tests
  - **WIP**: Work in progress, a temporary label for incomplete or ongoing work

# Changelog Entry

### Description

- i18n: update french translation fr-FR

---

### Additional Information

Hello! I added missing translations, fixed some errors, and rephrased some text in the current French translation. I will add comments in the MR diff to explain some of my changes.

I'd like to point out that I found some untranslated strings while browsing the app when testing my changes:

- "Create a knowledge base"
- "What are you working on?"
- "Name your knowledge base"
- "Search options"
- "a few seconds ago" (in admin user list)
- "Verify connection"
- "Server connection failed"
- "external" (in model list, while hovering the icon)
- "paragraphs"

### CODE REVIEW

File: src/lib/i18n/locales/fr-FR/translation.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
 	"(e.g. `sh webui.sh --api --api-auth username_password`)": "(par ex. `sh webui.sh --api --api-auth username_password`)",
 	"(e.g. `sh webui.sh --api`)": "(par exemple `sh webui.sh --api`)",
 	"(latest)": "(dernière version)",
-	"{{ models }}": "{{ modèles }}",
-	"{{ owner }}: You cannot delete a base model": "{{ propriétaire }} : Vous ne pouvez pas supprimer un modèle de base.",
+	"{{ models }}": "{{ models }}",
+	"{{ owner }}: You cannot delete a base model": "{{ owner }} : Vous ne pouvez pas supprimer un modèle de base.",
```
Comment (superzazu) [inline src/lib/i18n/locales/fr-FR/translation.json:7]: I suppose those strings `{{ models }}` should not be translated.

File: src/lib/i18n/locales/fr-FR/translation.json [hunk 2]
```diff
@@ -53,22 +53,22 @@
...
-	"and {{COUNT}} more": "",
+	"and {{COUNT}} more": "et {{COUNT}} autres",
 	"and create a new shared link.": "et créer un nouveau lien partagé.",
 	"API Base URL": "URL de base de l'API",
 	"API Key": "Clé d'API",
 	"API Key created.": "Clé d'API générée.",
 	"API keys": "Clés d'API",
 	"April": "Avril",
-	"Archive": "Archivage",
+	"Archive": "Archiver",
```
Comment (superzazu) [inline src/lib/i18n/locales/fr-FR/translation.json:63]: Some labels like this are actually verbs (button labels) and not nouns, so "Archiver" (verb) makes more sense than "Archivage" (noun).

File: src/lib/i18n/locales/fr-FR/translation.json [hunk 3]
```diff
@@ -201,10 +201,10 @@
...
 	"Didn't fully follow instructions": "N'a pas entièrement respecté les instructions",
 	"Disabled": "Désactivé",
-	"Discover a function": "Découvrez une fonction",
-	"Discover a model": "Découvrir un modèle",
-	"Discover a prompt": "Découvrir une suggestion",
-	"Discover a tool": "Découvrez un outil",
+	"Discover a function": "Trouvez une fonction",
+	"Discover a model": "Trouvez un modèle",
+	"Discover a prompt": "Trouvez un prompt",
+	"Discover a tool": "Trouvez un outil",
```
Comment (superzazu) [inline src/lib/i18n/locales/fr-FR/translation.json:207]: In sentences such as "Discover a model!", "Trouvez" ("Find") is a bit more natural than "Découvrez" in French.

File: src/lib/i18n/locales/fr-FR/translation.json [hunk 4]
```diff
@@ -285,28 +285,28 @@
...
 	"Export chat (.json)": "Exporter la conversation (.json)",
 	"Export Chats": "Exporter les conversations",
 	"Export Config to JSON File": "Exporter la configuration vers un fichier JSON",
-	"Export Functions": "Exportez des fonctions",
+	"Export Functions": "Exporter des fonctions",
 	"Export LiteLLM config.yaml": "Exportez le fichier LiteLLM config.yaml",
 	"Export Models": "Exporter des modèles",
 	"Export Prompts": "Exporter des prompts",
-	"Export Tools": "Outils d'exportation",
+	"Export Tools": "Exporter des outils",
```
Comment (superzazu) [inline src/lib/i18n/locales/fr-FR/translation.json:301]: This was an error, "Outils d'exportation" means "tools to export", and not "Export tools" (verb)

File: src/lib/i18n/locales/fr-FR/translation.json [hunk 5]
```diff
@@ -562,17 +562,17 @@
...
 	"Record voice": "Enregistrer la voix",
 	"Redirecting you to OpenWebUI Community": "Redirection vers la communauté OpenWebUI",
 	"Refer to yourself as \"User\" (e.g., \"User is learning Spanish\")": "Désignez-vous comme « Utilisateur » (par ex. « L'utilisateur apprend l'espagnol »)",
-	"References from": "",
+	"References from": "Références de",
 	"Refused when it shouldn't have": "Refusé alors qu'il n'aurait pas dû l'être",
 	"Regenerate": "Regénérer",
-	"Release Notes": "Notes de publication",
-	"Relevance": "",
+	"Release Notes": "Notes de mise à jour",
```
Comment (superzazu) [inline src/lib/i18n/locales/fr-FR/translation.json:574]: "Notes de publication" is not incorrect, but "Notes de mise à jour" (literally "Update notes") is a bit more natural, I suppose

File: src/lib/i18n/locales/fr-FR/translation.json [hunk 6]
```diff
@@ -627,10 +627,10 @@
 	"Searxng Query URL": "URL de recherche Searxng",
 	"See readme.md for instructions": "Voir le fichier readme.md pour les instructions",
 	"See what's new": "Découvrez les nouvelles fonctionnalités",
-	"Seed": "Graine",
+	"Seed": "Seed",
```
Comment (superzazu) [inline src/lib/i18n/locales/fr-FR/translation.json:630]: The term "seed" is heavily used in French (in IA / IT)

File: src/lib/i18n/locales/fr-FR/translation.json [hunk 7]
```diff
@@ -686,36 +686,36 @@
...
+	"Stop": "Stop",
 	"Stop Sequence": "Séquence d'arrêt",
 	"Stream Chat Response": "Streamer la réponse de la conversation",
 	"STT Model": "Modèle de Speech-to-Text",
 	"STT Settings": "Paramètres de Speech-to-Text",
 	"Subtitle (e.g. about the Roman Empire)": "Sous-titres (par ex. sur l'Empire romain)",
 	"Success": "Réussite",
 	"Successfully updated.": "Mise à jour réussie.",
-	"Suggested": "Sugéré",
+	"Suggested": "Suggéré",
```
Comment (superzazu) [inline src/lib/i18n/locales/fr-FR/translation.json:697]: Typo

File: src/lib/i18n/locales/fr-FR/translation.json [hunk 8]
```diff
@@ -725,37 +725,37 @@
...
+	"Toast notifications for new updates": "Notifications toast pour les nouvelles mises à jour",
 	"Today": "Aujourd'hui",
-	"Toggle settings": "Basculer les paramètres",
-	"Toggle sidebar": "Basculer la barre latérale",
-	"Token": "",
-	"Tokens To Keep On Context Refresh (num_keep)": "Jeton à conserver lors du rafraîchissement du contexte (num_keep)",
-	"Too verbose": "",
-	"Tool": "",
+	"Toggle settings": "Afficher/masquer les paramètres",
+	"Toggle sidebar": "Afficher/masquer la barre latérale",
```
Comment (superzazu) [inline src/lib/i18n/locales/fr-FR/translation.json:754]: "Afficher/masquer" (literally "Show/Hide") is much more used than the literal verb "Basculer"

**Comment by tjbck** [comment]
Thanks!

**Pull request merged**